playbulb-candle/app.js

```diff
@@ -3,11 +3,13 @@ var r = g = b = 255; // White by default.
 document.querySelector('#connect').addEventListener('click', function() {
   document.querySelector('#state').classList.add('connecting');
   playbulbCandle.connect()
-  .then(() => {
+  .then(_ => {
```

Comment:
Yes, that's exactly why. I've been using it since https://twitter.com/paul_irish/status/700477060196671488